timesketch/lib/analyzers/feature_extraction.py

```diff
@@ -229,7 +265,43 @@ def extract_feature(self, name, config):
                 continue
 
             event_counter += 1
-            event.add_attributes({store_as: result[0]})
+            #verify if store_as exist
+            store_type = event.source.get(store_as)
+            if store_type and not overwrite_store_as:
+                continue
+            if isinstance(store_type, six.text_type):
+                store_type_list = False
+            elif isinstance(store_type, (list, tuple)):
+                store_type_list = True
+            if store_type_list:
+                if keep_multimatch:
+                    if store_type and overwrite_and_merge_store_as:
+                        #don't add duplicate value
+                        resultList = list(set(store_type) | set(result))
```

Comment:
we use snake case for variables, so this should be `result_list`

Reply:
ok